newdle/client/src/components/creation/timeslots/Timeline.module.scss

```
@@ -182,6 +181,21 @@ $label-width: 180px;
   }
 }
 
+@media screen and (min-width: 1200px) {
+  .timeline {
+    margin: 4px;
+    margin-left: $label-width;
+    @include timeline-all;
+  }
+}
+
+@media screen and (max-width: 1199px) {
+  .timeline {
+    margin: 4px;
+    @include timeline-all;
+  }
+}
+
```

You can simplify this:

```suggestion
.timeline {
  margin: 4px;
  @include timeline-all;
}

@media screen and (min-width: 1200px) {
  .timeline {
    margin-left: $label-width;
  }
}
```